gpkit/repr_conventions.py

```diff
@@ -14,3 +15,16 @@ def _str(self):
 def _repr_latex_(self):
     "Returns default latex for automatic iPython Notebook rendering."
     return "$$"+self.latex()+"$$"
+
+
+def unitstr(units, into="%s", options="~", dimless=""):
+    "Returns the string corresponding to an object's units."
+    if hasattr(units, "units") and isinstance(units.units, Quantity):
+        units = units.units
+    if not isinstance(units, Quantity):
+        return dimless
+    rawstr = (u"{:%s}" % options).format(units.units)
+    if str(units.units) == "count":
```

Comment:
My point is that above you have `units = units.units` inside a conditional `if hasattr(units, "units")`. So I'm led to believe that `units` might not even have a `units` attribute. But then we do `if str(units.units) == "count":`. Why not `if str(units) == "count":`?